example/_config.yml

```diff
@@ -8,6 +8,9 @@ description: > # this means to ignore newlines until "baseurl:"
 baseurl: "/minima"
 twitter_username: jekyllrb
 github_username:  jekyll
+# if you want to use disqus as comment system, be aware that it only works on production
+disqus:
+  shortname: your_disqus_shortname
```

Comment:
I still feel there's no place for `production`-only features to be included in the `example/` as long as the example is going to be served only locally. though, I hv never tried disqus with `//localhost`